app/models/match_decisions/homeless_set_aside/confirm_hsa_accepts_client_decline_dnd_staff.rb

```diff
@@ -0,0 +1,97 @@
+module MatchDecisions::HomelessSetAside
+  class ConfirmHsaAcceptsClientDeclineDndStaff < ::MatchDecisions::Base
+
+    def statuses
+      {
+        pending: 'Pending',
+        decline_overridden: 'Decline Overridden',
+        decline_overridden_returned: 'Decline Overridden, Returned',
+        decline_confirmed: 'Decline Confirmed',
+        canceled: 'Canceled',
+       }
+    end
+
+    def label
+      label_for_status status
+    end
+
+    def label_for_status status
+      case status.to_sym
+      when :pending then "#{_('DND')} to confirm match success"
```

Comment:
This isn't correct.  should be `"#{_('DND')} to confirm #{_('Housing Subsidy Administrator')} decline`

Reply:
Fixed both here and on the provider only route.